app/controllers/downloads_controller.rb

```diff
@@ -48,14 +48,6 @@ def download
       @download64portable = Download.latest_for(@platform + "64Portable")
       @latest = Version.latest_version
 
-      if request.env["HTTP_USER_AGENT"] =~ /WOW64|Win64/
-        @download = @download64
-        @bitness = "64-bit"
-      else
-        @download = @download32
```

Comment:
these `download` and `bitness` variables are passed to the template. Since they are not defined now, we see the ` (#{@download32.version.name})` in the rendered page.

Reply:
we are not using `download` and `bitness` **ruby** variables in the template anymore

Reply:
haven't dig enough to be sure about this, but don't we need the download variable to start the download variable automatically? (I'll find sometime today to have a deeper look on this)

Reply:
@pedrorijo91 Before you were detecting 'bitness' and the correct version to download on the server side, but I've changed it in a way so everything now depends on the client side. You can take a look at another file in the commit and see how it <s>is</s> (should) work now. Download link is also detected on the client side and then placed in an iframe as before.